src/credentials/sso/authorization.ts

```diff
@@ -0,0 +1,13 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export interface Authorization {
```

Comment:
Ditto with above and maybe `SsoAuthorization`

Reply:
Interface was not needed.  Deleted.